python/lsst/obs/subaru/objectMasks.py

```diff
@@ -0,0 +1,132 @@
+import re
+import lsst.daf.base as dafBase
+import lsst.pex.logging as pexLog
+import lsst.afw.coord as afwCoord
+import lsst.afw.geom as afwGeom
+import lsst.afw.table as afwTable
+
+class ObjectMaskCatalog(object):
+    """Class to support bright object masks
+
+    N.b. I/O is done by providing a readFits method which fools the (old) butler.
+    """
+    def __init__(self):
+        schema = afwTable.SimpleTable.makeMinimalSchema()
+
+        schema.addField("radius", 'Angle', "radius of mask")
+        self._catalog = afwTable.SimpleCatalog(schema)
+        self._catalog.table.setMetadata(dafBase.PropertyList())
+
+        self.table  = self._catalog.table
+        self.addNew = self._catalog.addNew
+
+    def __len__(self):
+        return len(self._catalog)
+
+    def __iter__(self):
+        return iter(self._catalog)
+
+    def __getitem__(self, i):
+        return self._catalog.__getitem__(i)
+
+    def __setitem__(self, i, v):
+        return self._catalog.__setitem__(i, v)
+
+    @staticmethod
+    def readFits(fileName, hdu=0, flags=0):
+        """Read a ds9 region file, returning a ObjectMaskCatalog object
+
+        This method is called "readFits" to fool the butler. The corresponding mapper entry looks like
+        brightObjectMask: {
+            template:      "deepCoadd/BrightObjectMasks/%(tract)d/BrightObjectMask-%(tract)d-%(patch)s-%(filter)s.reg"
+            python:        "lsst.obs.subaru.objectMasks.ObjectMaskCatalog"
+            persistable:   "PurePythonClass"
+            storage:       "FitsCatalogStorage"
+        }
+        and this is the only way I know to get it to read a random file type, in this case a ds9 region file
```

Comment:
There's no inherent FITS bias.  There's just a hack that was added to allow FITS catalogs before a general plugin framework could be developed that happens to provide a way to load arbitrary Python objects.  The principled way to do this currently is (given that a C++ `Storage` subclass is undesirable for a Python object) to add a new Storage to the switch in `daf_persistence:butler.py`.  I think this will actually be a relatively high priority item to enable new types of objects and ways of serializing and persisting them.